docs/csharp/programming-guide/classes-and-structs/local-functions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Local functions (C# Programming Guide) | Microsoft Docs"
+ms.date: "2017-06-14"
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "fields [C#]"
```

Comment:
I'm not sure what `helpviewer_keywords` are exactly, but I don't think it should be "fields" here.